eclipse/plugin.xml

```diff
@@ -1469,12 +1469,4 @@
          </adapter>
       </factory>
    </extension>
-   <extension
```

Comment:
This is not BUILD related.

Reply:
True, I will move the change into another commit.